_rules/html-page-lang-xml-lang-match-5b7ae0.md

```diff
@@ -48,7 +48,7 @@ Since most assistive technologies will consistently use `lang` over `xml:lang` w
 
 - [H57: Using language attributes on the html element](https://www.w3.org/WAI/WCAG21/Techniques/html/H57)
 - [BCP 47: Tags for Identifying Languages](https://www.ietf.org/rfc/bcp/bcp47.txt)
-- [The `lang` and `xml:lang` attributes](https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes)
+- [The `lang` and `xml:lang` attributes](https://www.w3.org/TR/html52/dom.html#the-lang-and-xmllang-attributes)
```

Comment:
I don't get it. Why are we citing HTML 5.2 again? Didn't we decide to move to WHATWG?
I guess we haven't fully resolved #879.

Reply:
I am always unsure about this. We should update our test to catch this too then.